include/grpcpp/support/channel_arguments_impl.h

```diff
@@ -31,6 +31,15 @@ namespace grpc {
 namespace testing {
 class ChannelArgumentsTest;
 }  // namespace testing
+
+namespace experimental {
+/// Validates \a service_config_json. If valid, returns an empty string.
+/// Otherwise, returns the validation error.
+/// TODO(yashykt): Promote it to out of experimental once it is proved useful
+/// and gRFC is accepted.
+grpc::string ValidateServiceConfigJSON(const grpc::string& service_config_json);
```

Comment:
I'm not sure this file is the right place to put this, since it really has nothing to do with channel args.  But I'm not sure there's a better place.

Vijay, do you have any suggestions on the right place to put this?

Reply:
This only gets fed to a ChannelArguments method, correct? It seems like it should better be a static method of that class rather than a free function then.

Reply:
The setter for the default ServiceConfigJSON does get fed through channel arguments, but whether a service config is valid or not is not really a property of ChannelArguments. This function would be potentially be used by unit tests.

Reply:
Reasonable to leave it a free function then, but it should preferably get its own header file - there's no cost to make new ones.